src/countItems.js

```diff
@@ -11,12 +11,20 @@
  * @returns {object} An object containing the count of each
  *  string in the input array
  */
-function countItems(array) {
-  // This is your job. :)
+function countItems(array,value) {
```

Comment:
This function returns how many times `value` appears in `array`, but that's not what I'm asking you to do. You don't need to modify the function definition of any of the functions I provided.

See the `@example` section in the comment above the function. It gives an example input and example output, as well as a plain-English description of what the question is asking.

If it's not clear, ping me on Discord.